react/Logo/Logo.js

```diff
@@ -15,6 +15,7 @@ const viewportDimensions = type => ({
 const getViewPortDimension = (type, dimension) => viewportDimensions(type)[dimension];
 
 export default function Logo({
+  locale,
```

Comment:
FYI I had to add this so we don't get warnings about passing `locale` to the underlying `div`.